content/documentation/v1.37/configuration/vm.adoc

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Wasn't exactly sure where to put this info. Also considered putting it under FAQ since it's less about configuring Kiali and more about configuring Prometheus in order to get the telemetry from the VMs for Kiali to display.